pkg/release/release.go

```diff
@@ -406,14 +433,14 @@ func (r *Release) upgrade(client helm.Client, hr *v1.HelmRelease, chart chart, v
 		ResetValues: !hr.GetReuseValues(),
 		SkipCRDs:    hr.Spec.SkipCRDs,
 		MaxHistory:  hr.GetMaxHistory(),
-		Wait:        hr.Spec.Wait || hr.Spec.Rollback.Enable,
```

Comment:
existing behavior is that wait only defaults to true on upgrades (not installs) when rollbacks are enabled. It seemed better to me for the config to be statically determined, and not depend on current action (install vs upgrade), so I implemented that for when tests are enabled, and then I just made the behavior match when rollbacks are enabled. This does mean wait may be turned on for some installs where it wasn't before. Let me know what you think.

Reply:
This is :ok_hand:, I would have made the same change if I had to further extend the waiting behaviour.